### DISCUSSION TITLE
Mobile: Scrolling long starred lists causes refresh that creates duplicates and omissions

### DISCUSSION THREAD

**Discussion by boldandbrad** [community/community#117155] [Other Feature Feedback, Questions, & Ideas]
Scrolling down on a long (~20+ entries) starred list forces the view to reload, removing many of the entries and often showing duplicates of the ones that remain. 

This makes it difficult, if not impossible, to find repos that have been categorized in lists - defeating the feature's purpose.

**Comment by Ahmed1na** [comment]
01110123883

**Reply by Gedoxa** [reply]
Čččė
